docs/installation/kepler-operator.md

```diff
@@ -25,26 +26,34 @@ prometheus-operator-6bd88c8bdf-9f69h   2/2     Running   0          48m
 ```
 
 ## Run kepler-operator
+
 - You can use the image from quay.io to deploy kepler-operator.
 
 ```sh
-make deploy IMG=quay.io/sustainable_computing_io/kepler-operator:latest
-kubectl config set-context --current --namespace=monitoring
+make deploy OPERATOR_IMG=quay.io/sustainable_computing_io/kepler-operator:latest
```

Comment:
`make fresh` already builds the operator, pushes it to local repo. Devs just need to run the `operator-sdk run bundle` command.